pkg/cli/cli_handle.go

```diff
@@ -52,19 +52,19 @@ Usage: {{ COMMAND }} [options]
 
 Handle "do" commands in the CLI request YAML file at the given path:
 
-      aod cli handle -path "/path/to/file.yaml"
+      aod cli {{ COMMAND }} -path "/path/to/file.yaml"
```

Comment:
`COMMAND` is the _full_ command

```suggestion
      {{ COMMAND }} -path "/path/to/file.yaml"
```